contacts-in-mautic.php

```diff
@@ -33,6 +33,12 @@
 if ( ! defined( 'WPINC' ) ) {
 	die;
 }
+// Set the BSF_CONTACT_MAUTIC_VERSION.
+define('BSF_CONTACT_MAUTIC_VERSION', '1.0.3');
```

Comment:
@nileshc-bsf This version is different than the version defined in comments in main plugin file - https://cl.ly/f535fe63c6d1 - Please check!

Reply:
Version number updated to 1.0.2